grails-app/views/species/_literature.gsp

```diff
@@ -16,16 +16,15 @@
         <div class="col-md-9" style="padding-top:14px;">
 
             <div id="bhl-integration">
-                <h3>Name references found in the <a href="http://biodiversityheritagelibrary.com/" target="_blank">Biodiversity Heritage Library</a></h3>
+                <h3>Name references found in the <a href="${grailsApplication.config.literature.bhl.url}/search?SearchTerm=%22${synonyms?.join('%22+OR+%22')}%22&SearchCat=M#/names" target="_blank">Biodiversity Heritage Library</a></h3>
                 <div id="bhl-results-list" class="result-list">
-                    <a href='http://www.biodiversitylibrary.org/search?SearchTerm=%22${synonyms?.join('%22+OR+%22')}%22&SearchCat=M#/names' target='bhl'>Search BHL for references to ${tc?.taxonConcept?.nameString}</a>
```

Comment:
Not familiar with the code, but is this div automatically filled by a browser script?

Reply:
Yes. I've added in a comment to clarify this. The link was inconsistent with the trove behaviour below, so I've made them consistent.